react/InputPassword/styles.styl

```diff
@@ -0,0 +1,8 @@
+.wrapInputPassword
```

Comment:
We use to use dash-case for or CSS class names. We are also prefixing or component related class names with `c-`.

Actually it was not in our guidelines, I juste made a PR to fix that. https://github.com/cozy/cozy-guidelines/pull/57

Reply:
https://docs.cozy.io/cozy-ui/styleguide/ there is guidelines but not in cozy-guidelines I admit.

Reply:
@GoOz is it good now ?

Reply:
where is the link?

Reply:
[new styles.styl](https://github.com/cozy/cozy-ui/blob/fc7206c7b78148dffc48ad519e77f8a042717513/react/Field/styles.styl)
I add the classes : `.wrap-input-password` and `.action-on-input`  :)